exe/kubernetes-deploy

```diff
@@ -68,8 +68,8 @@ end
 template_paths = [template_dir] if template_paths.empty? && template_dir
 
 begin
-  KubernetesDeploy::OptionsHelper.with_processed_template_paths(template_paths) do |paths|
-    runner = KubernetesDeploy::DeployTask.new(
+  Krane::OptionsHelper.with_processed_template_paths(template_paths) do |paths|
+    runner = Krane::DeprecatedDeployTask.new(
```

Comment:
Wait, is `KubernetesDeploy::DeployTask` (and other tasks) still gone? We can't do that yet. That's part of the public interface, and we decided not to break that in the prereleases.

Reply:
Yup, addressing this right now! Thanks again for your insights!